finagle-base-http/src/main/scala/com/twitter/finagle/http/param/params.scala

```diff
@@ -64,6 +64,12 @@ object Streaming {
     Stack.Param(Streaming(enabled = false))
 }
 
+case class FixedLengthStreamedAfter(size: StorageUnit)
+object FixedLengthStreamedAfter {
+  implicit val fixedLengthStreamedAfter: Stack.Param[FixedLengthStreamedAfter] =
+    Stack.Param(FixedLengthStreamedAfter(StorageUnit.zero))
```

Comment:
@luciferous talked about using 8k for a default value here. I liked that at first but maybe we're better off preserving the current "default" behavior and using 5.mb as a value?

Reply:
Sounds good.

Reply:
@edio This thread.